src/main/java/io/cdap/plugin/gcp/firestore/common/FirestoreConfig.java

```diff
@@ -10,7 +10,6 @@
 import io.cdap.cdap.etl.api.FailureCollector;
 import io.cdap.plugin.common.Constants;
 import io.cdap.plugin.common.IdUtils;
-
```

Comment:
please revert this change in all the files, java imports are meant to be kept separate I think otherwise build will fail.

Reply:
Fixed.